### ISSUE TITLE
Wrong Docker Server version

### ISSUE THREAD

**Issue by Trojan295** [void-linux/void-packages#2221]
Hello,

I noticed that `docker version` prints that the Docker Server version is being `dev`. Due to this I'm having problems when using [sbt-docker](https://github.com/marcuslonnberg/sbt-docker) in Void Linux.

```
[error] java.lang.RuntimeException: Could not parse Version from dev
[error] : non-negative integer value expected
```

### System

* xuname:  Void 4.17.19_1 x86_64 GenuineIntel uptodate rrrm
* package:  docker-18.06.0_2

### Expected behavior
```
$ docker version
Client:
 Version:      17.06.0-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:15:15 2017
 OS/Arch:      linux/amd64
Server:
 Version:      <proper version, eg. 18.06.0-ce>
 API version:  1.38 (minimum version 1.12)
 Go version:   go1.11
 Git commit:   v18.06.0-ce
 Built:        Mon Aug 27 00:02:50 2018
 OS/Arch:      linux/amd64
 Experimental: false
```
### Actual behavior
```
$ docker version
Client:
 Version:      17.06.0-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:15:15 2017
 OS/Arch:      linux/amd64
Server:
 Version:      dev
 API version:  1.38 (minimum version 1.12)
 Go version:   go1.11
 Git commit:   v18.06.0-ce
 Built:        Mon Aug 27 00:02:50 2018
 OS/Arch:      linux/amd64
 Experimental: false
```
### Steps to reproduce the behavior

`$ docker version`

**Comment by Trojan295** [comment]
I spotted the problem: the env variable VERSION is missing when running make.sh and it builds with the default `dev`.
I will send a PR for this.

**Comment by Hoshpak** [comment]
fixed by #2139
